bin/run_multiple_panels.py

```diff
@@ -0,0 +1,465 @@
+import os
+import shutil             
+import sys
+import time
+import datetime as dt
+import subprocess
+from multiprocessing import Pool
+import pandas as pd
+
+from ochre import Dwelling, Analysis, ElectricVehicle
+
+# Script to run multiple simulations. Assumes each simulation has a unique folder with all required inputs
+
+# Download weather files from: https://data.nrel.gov/submissions/156 or https://energyplus.net/weather
+# weather_path = os.path.join('path', 'to', 'weather_files')
+weather_path = os.path.join(os.getcwd(), 'ResStockFiles', 'Weather')
+
+end_use_names=['HVAC Heating Electric Power (kW)', 'HVAC Cooling Electric Power (kW)', 'Water Heating Electric Power (kW)',
```

Comment:
Do we need pool/spa pumps/heaters on this list? I was leaning towards not including those because our model isn't up to snuff, let me confirm with LBNL next time we talk.

Reply:
The only place this is used is to calculate the whole home amperage, where I think it deserves a discussion whether to monitor whole home power or whole home current. I think we might have discussed this last year but it's worth revisiting.

Reply:
In that case, don't we need to include all the uncommon electrical loads? Things like well pumps and freezers are only in a small % of homes but definitely affect whole home power.

If you don't already have it I can get you a list of all the ones that'd come up in ResStock.

Reply:
Yes, we should include all loads to make this general. I included these based on what loads there were for the Vermont homes that we simulated last year, but it's not an exhaustive list.

Reply:
Here's everything that can show up but is in a small # of homes:

Extra Refrigerator
Freezer
Hot Tub Spa
Pool Heater
Pool Pump
Well Pump

If it's helpful I can also generate a .xml file for a home with all of these for testing through BEopt.

There's a few uncommon gas loads as well, like gas lighting. Not relevant here but just FYI.